Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM registry.opensuse.org/opensuse/leap:latest
+
+ENV PYTHONDONTWRITEBYTECODE=1 PYTHONUNBUFFERED=1 UWSGI_WSGI_FILE=/pcw/webui/wsgi.py UWSGI_MASTER=1
+ENV UWSGI_HTTP_AUTO_CHUNKED=1 UWSGI_HTTP_KEEPALIVE=1 UWSGI_LAZY_APPS=1 UWSGI_WSGI_ENV_BEHAVIOR=holy
+
+## System preparation steps ################################################# ##
+
+# Install system requirements
+RUN zypper -n in python3 python3-devel python3-pip gcc
+## Install SUSE CA. Comment the following two lines, if this is not desired
+RUN zypper -n ar http://download.suse.de/ibs/SUSE:/CA/openSUSE_Leap_15.3/ SUSE_CA
+RUN zypper -n in ca-certificates-suse
+COPY requirements.txt /
+# Install pip requirements
+RUN pip install -r requirements.txt
+# The cache (especially zypper) is not required for the resulting container
+RUN rm -rf /var/cache /requirements.txt
+
+## Prepare pcw ############################################################## ##
+
+COPY . /pcw/
+RUN rm -rf /pcw/.git
+RUN rm -rf /pcw/db
+
+# Create links to config files and database directory on the /data volume
+RUN ln -s /data/pcw.ini /etc/pcw.ini
+RUN ln -s /data/db /pcw/db
+# Create /pcw/pcw for convenience
+RUN ln /pcw/container-startup /pcw/pcw
+# pcw assumes the static content to be located in /pcw/static
+RUN ln -s /pcw/ocw/static /pcw/static
+
+## Finalize ################################################################# ##
+
+VOLUME /data
+EXPOSE 8000/tcp
+
+WORKDIR /pcw
+# Once we are certain that this runs nicely, replace this with ENTRYPOINT.
+CMD ["/pcw/pcw", "run"]
```

Comment:
See lines 28-29 in the Dockerfile

    # Create /pcw/pcw for convenience
    RUN ln /pcw/container-startup /pcw/pcw

The reason to link this is that if one needs to run custom `pcw` commands (e.g. add admin emails, migrate ...) this can be done with a nice-to-remember path, instead of performing the whole `python migrate.py runserver` magic.

Reply:
This also allows to easily call additional commands, e.g. `createsuperuser` (see the next comment).